model/ValidationService.php

```diff
@@ -39,7 +39,7 @@ class ValidationService extends ConfigurableService
             '/qti/data/apipv1p0final/Core_Level/Package/apipv1p0_qtiitemv2p2_v1p0.xsd'
         ],
         'http://www.imsglobal.org/xsd/imsqti_v2p2' => [
-            '/qti/data/qtiv2p2p1/imsqti_v2p2p1.xsd'
+            '/qti/data/qtiv2p2p4/imsqti_v2p2p4.xsd'
```

Comment:
do we need @julien-sebire to confirm this ?

Reply:
This is correct to my understanding.
In parallel, the tech debt reduction ticket https://oat-sa.atlassian.net/browse/CGF-264 has been created, so that we stop duplicating this.